kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/MongoKVTable.java

```diff
@@ -68,6 +71,7 @@ public MongoKVTable(
       final CollectionCreationOptions collectionCreationOptions
   ) {
     this.name = name;
+    this.keyCodec = new StringKeyCodec();
```

Comment:
let's add comments re: why we decided to add this and why we're not using byte[]